apps/dhparam.c

```diff
@@ -314,7 +302,6 @@ int dhparam_main(int argc, char **argv)
 
     if (check) {
         if (!EVP_PKEY_param_check(ctx) /* DH_check(dh, &i) */) {
-            ERR_print_errors(bio_err);
             BIO_printf(bio_err, "ERROR: Invalid parameters generated\n");
```

Comment:
For consistency, would you mind changing `ERROR:` to `Error,`?

Reply:
Good point, done as well, also for the 2nd such thing.